index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Who's the MVP?</title>
+</head>
+<body bgcolor="#BCFCFA">
+	<h1>Who will be the NBA's Most Valuable Player?</h1>
+	<table style="margin-left: 60px;">
```

Comment:
Tables are useful for representing tabular data, not for laying out the page anymore. They were very popular in the early web. Now you can use the HTML5 layout tags to accurately label your page sections. 

html5doctor.com/downloads/h5d-sectioning-flowchart.png